.github/workflows/commit_manifest.yml

```diff
@@ -3,6 +3,10 @@ on:
   push:
     branches:
       - "main"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
I think `cancel-in-progress: true` is correct. But to be explicit, if we merged three PRs in a row only the last merged PR would publish a manifest. I can't think of why we would need any of the previous manifests so this should be ok.

Reply:
yeah, this is what i was wanting to see happen, i think we keep